### PR TITLE
Fix template attributes forbidden

### LINKED ISSUE #136: twig 2 inheritance changes with macro - Accessing Twig_Template attributes is forbidden.
Related Issue #106 

This Issue is still present. A PR was Merged into development but never made it into master. 

https://github.com/hackzilla/TicketBundle/commit/89d3a9226283548ac9f0d210aa2d9e4a4fa5f61b

should be merged into master.

### PR DESCRIPTION
**Pull request by masonen**
This fixes #136 (and #106 )

The fix was pulled into development but never merged into master.

### CODE REVIEW

File: Resources/views/Ticket/show_attachment.html.twig [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{% import 'HackzillaTicketBundle:Macros:macros.html.twig' as macros %}
```
Comment (phansys) [inline Resources/views/Ticket/show_attachment.html.twig]: ec233c90bb2cdba896031bae541f51b157694abb is now cherry-picked into `master`, thanks for pointing the issue.
```suggestion
{% import '@HackzillaTicket/Macros/macros.html.twig' as macros %}
```

**Comment by masonen** [comment]
changed it in another way. index.html.twig imports macros in the same way. If you do it that way macros could possibly be overidden without needing to override show_attachment.html.twig

**Comment by benjaminmeissner** [comment]
@phansys what do you think?

**Comment by masonen** [comment]
@phansys what do you think

**Comment by phansys** [comment]
I'm not checked the proposed solution yet @masonen. Sorry for the delay, I'll try to give it a chance ASAP.
Thank you in advance.

**Comment by phansys** [comment]
This PR was squashed before being merged into the `master` branch at @03df9d4a116d3debdc8962f617692b1045ad2971.

**Comment by phansys** [comment]
Thank you @masonen.

**Pull request closed without merging**